lms/templates/courseware/accordion.html

```diff
@@ -19,15 +19,15 @@
 <a href="#${chapter['display_id']}-child" role="button" class="button-chapter chapter ${active_class}" id="${chapter['display_id']}-parent" aria-controls="${chapter['display_id']}-child" aria-expanded="false">
     <span class="group-heading ${active_class}" aria-label="${aria_label}">
         <span class="icon fa fa-caret-right" aria-hidden="true"></span>
-        ${chapter['display_name']}
+        ${HTML(chapter['display_name'])}
     </span>
 </a>
 <div class="chapter-content-container" id="${chapter['display_id']}-child" tabindex="-1" role="region" aria-label="${chapter['display_name']} submenu">
```

Comment:
@attiyaIshaque Is there any possibility that screen readers failing to read improperly escaped aria-label?